scripts/ssb-web/uninstall

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+sudo systemctl stop ssb-web-broadcast.service || true
+sudo systemctl disable ssb-web-broadcast.service || true
+sudo systemctl daemon-reload
+
+sudo rm -rf /var/www/backend
+sudo rm -rf /var/www/sbot
+sudo rm -rf /usr/lib/node_modules
+sudo rm -rf /var/www/.ssb
+sudo rm -f /usr/local/bin/ssb-web-broadcast-service.sh
+
+sudo rm -rf /etc/nginx/site-path-enabled/ssb-web.conf
+
+sudo npm uninstall -g --unsafe-perm \
```

Comment:
Do we need sudo sudo here too? In case it's trying to remove root owned folders after dropping out of sudo? Or does that happen with post-**install** scripts only.

Reply:
so far its post-install. Well see if its a problem